go.mod

```diff
@@ -100,6 +100,7 @@ require (
 	github.com/nats-io/nats.go v1.13.1-0.20211018182449-f2416a8b1483
 	github.com/newrelic/newrelic-telemetry-sdk-go v0.5.1
 	github.com/nsqio/go-nsq v1.1.0
+	github.com/olivere/elastic v6.2.37+incompatible
```

Comment:
Is there a reason we need to update the version of elastic for this PR?

Reply:
i mean technically looking at the source of the older version, it does have the pipeline input..  This probably happened in different ways of testing solutions to this problem.. 

With that said, the older version was released Jan 2018, Elastic 7 full support wasn't added until March 2019.

Reply:
This is marked as outdated due to another PR: #10513